src/app/shared/filesystem/file-browser/file-browser.component.ts

```diff
@@ -187,7 +187,7 @@ export class FileBrowserComponent implements OnInit, OnDestroy {
    * @param {MouseEvent} $event
    * @memberof FileBrowserComponent
    */
-  emitContainerClick($event: MouseEvent): void {
+  emitContainerClick($event: any): void {
```

Comment:
Why are these typed as `any` now?

Reply:
This was a remnant of another implementation I tried. Must have been merged back in somewhere. Resolved with d68be92.